.github/workflows/test.yml

```diff
@@ -9,18 +9,13 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    container:
+      image: composer
```

Comment:
can you add a version tag here. Otherwise it will always use `latest`, which can change

looking at docker hub, looks like you can use `composer:1.9`

Reply:
Sure! Sorry, I was going to and forgot 🤦‍♂ Thanks! Done!